Messenger/Authorization/Network/AuthNetworkManager.swift

```diff
@@ -13,234 +13,230 @@ class AuthorizationNetworkManager: NetworkManager {
     
     let router = Router<AuthApi>()
     
-    func beforeLogin(email: String, completion: @escaping (MailExistsResponse?, String?)->()) {
+    func beforeLogin(email: String, completion: @escaping (MailExistsResponse?, String?, Int?)->()) {
```

Comment:
You don't need to send response code to viewmodel. Send error as error type. i.e authorization error